Rover/RC_Channel.h

```diff
@@ -32,6 +32,7 @@ class RC_Channels_Rover : public RC_Channels
 
 public:
 
+    bool in_rc_failsafe() const override;
     bool has_valid_input() const override;
```

Comment:
I suspect the existing has_valid_input() could be moved to common code now.

Reply:
out of scope